pkg/tekton/syntax/pipeline.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// GitMergeImage is the default image name that is used in the git merge step of a pipeline
-	GitMergeImage = "gcr.io/jenkinsxio/builder-jx:0.1.477"
+	GitMergeImage = "gcr.io/jenkinsxio/builder-jx:0.1.527"
```

Comment:
I would be great if we could move this out form the code into https://github.com/jenkins-x/jenkins-x-versions/tree/master/docker. It would make the update easier.

Reply:
Do you have a pointer on how to do this? Not sure how to do it.

Reply:
Wondering if we should merge this in the meantime, given that the 0.1.477 version is broken for merging.

Reply:
https://github.com/jenkins-x/jx/blob/d5f7e09a56a7863ab5ae0c8cbb093f5b3c8797a6/pkg/cmd/step/create/step_create_task.go#L222

Reply:
I think is fine to merge it. You can move it out in a follow up PR.

Reply:
LOL, ahh that's what this is. I was wondering what this VersionResolver is, but never dived into it. That seems straight forward. There is already `docker/gcr.io/jenkinsxio/builder-jx.yml`. So really I just need to get this version and of course, ensure that version stream contains a recent enough version of builder-jx. 

For the latter, do i just need to create a pull request in the versions repo?

Reply:
yes, if you just introduce a new docker image in the version stream is no problem to merge the PR straight away.